eox_nelp/course_experience/models.py

```diff
@@ -94,6 +96,25 @@ class Meta:
         """Set model abstract"""
         abstract = True
 
+    def save(self, *args, **kwargs):
+        """Overrides save method in order to add extra functionalities."""
+        super().save(*args, **kwargs)
+
+        self.emit_feedback_event()
```

Comment:
The event is not valid if the save() method fails since the data is not in the DB that's why the emit event must be after 
Save method doesn't have any return https://github.com/django/django/blob/main/django/db/models/base.py#L767

Personally I consider the signal approach a bad practice 

https://stackoverflow.com/a/35950538
https://seddonym.me/2018/05/04/django-signals/
https://lincolnloop.com/insights/django-anti-patterns-signals/
https://www.quora.com/Are-Django-signals-an-anti-pattern